src/main/kotlin/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,35 @@
+package lotto.model
+
+data class Lotto(val lottoNumbers: List<LottoNumber>) {
+    init {
+        require(isUniqueSixNumbers(lottoNumbers)) { "로또 번호는 서로 다른 6개의 숫자여야만 합니다." }
+    }
+
+    constructor(numbers: Collection<Int>) : this(numbers.map(::LottoNumber))
+
+    constructor(stringNumbers: String) : this(splitByComma(stringNumbers))
+
+    fun getWinningCount(winningNumbers: List<Int>): Int {
+        return lottoNumbers.filter { winningNumbers.contains(it.number) }.count()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return lottoNumbers.count { winningNumbers.contains(it.number) }
```